packages/databricks-vscode/resources/whats-new/1.2.md

```diff
@@ -0,0 +1,48 @@
+# <!-- DATABRICKS LOGO --> Databricks Extension v1.2
+
+## What's new?
+
+-   [📗 Advanced notebook support with Databricks Connect V2](#dbconnect-notebook)
+
+## <a id="dbconnect-notebook"></a> Advanced notebook support with Databricks Connect V2
+
+You could already run notebooks locally using Databricks Connect V2. But we are now introducing advanced notebook support which enables you to run notebooks locally with the similar experience as running them on a Databricks cluster. The aim of this update is to bring most of the convenience of running notebooks on a Databricks cluster to your local environment!
+
+This includes:
+
+-   [Magic commands such as `%run` and `%sql` etc](#magic-commands)
+-   [Preconfigured globals such as spark and dbutils](#preconf-globals)
+-   [Interactive `dbutils.widgets`](#widgets)
+-   [Support for Databricks Notebooks](#dbnb)
```

Comment:
```suggestion
This update includes:

-   [Magic commands such as `%run` and `%sql`](#magic-commands)
-   [Preconfigured globals such as `spark` and `dbutils`](#preconf-globals)
-   [Interactive widgets through `dbutils.widgets`](#widgets)
-   [Support for Databricks notebooks](#dbnb)
```

Reply:
`spark` and `dbutils` are programmatic constructs, hence the backticks. 

"notebooks" should not be capitalized.